productMods/templates/freemarker/edit/forms/addProjectUpdate.ftl

```diff
@@ -0,0 +1,186 @@
+<#assign modificationNoteTextValue = lvf.getFormFieldValue(editSubmission, editConfiguration, "modificationNoteText") />
+<#assign modifiedByUriValue = lvf.getFormFieldValue(editSubmission, editConfiguration, "modifiedByUri") />
+<#assign modifiedOnValue = lvf.getFormFieldValue(editSubmission, editConfiguration, "modifiedOn") />
+
+
+<h2 xmlns="http://www.w3.org/1999/html">${formTitle}</h2>
+<form class="editForm customForm" id="addProjectUpdate" method="post" enctype="multipart/form-data" action="${submitUrl}">
+
+    <p>
+        <input type="hidden" name="editKey" id="editKey" value="${editConfiguration.editKey}" role="input">
+        <input type="hidden" name="subjectUri" id="subjectUri" value="${editConfiguration.subjectUri}">
+    </p>
+
+    <p>
+        <label for="title">${i18n().project_update_title} ${requiredHint}:</label>
+        <input type="text" name="title" id="title" label="title" size="50" role="input" value="${titleValue}">
+    </p>
+
+    <p>
+        <label for="reportingYear">${i18n().reporting_year}:</label>
+        <#assign reportingYearOpts = editConfiguration.pageData.reportingYearUri />
+        <select name="reportingYearUri" id="reportingYearUri" >
+            <option value="" <#if reportingYearValue = "">selected</#if>>${i18n().select_one}</option>
+        <#list reportingYearOpts?keys as key>
+            <option value="${key}" <#if reportingYearValue = key>selected</#if>>${reportingYearOpts[key]}</option>
+        </#list>
+        </select>
+    </p>
+
+    <p>
+        <label for="updateText">${i18n().project_update_text}:</label>
+        <textarea rows="10" cols="50" name="updateText" id="updateText" class="useTinyMce" role="textarea">${updateTextValue}</textarea>
+    </p>
+
+    <p>
+        <label for="publication">${i18n().associated_publication}:</label>
+        <input class="acSelector" size="60"  type="text" id="publicationTitle" name="publicationTitle" acGroupName="publication"  value="" />
+        <a href="#publicationTitleClear" class="clear" name="publicationTitleClear" id="publicationTitleClear"> ${i18n().clear_link}</a>
+    </p>
+
+    <div class="acSelection" acGroupName="publication" id="pubAcSelection">
+        <p class="inline">
+            <label>${i18n().selected_publication}:</label>
+            <span class="acSelectionInfo"></span>
+            <br />
+            <a href="" class="verifyMatch"  title="${i18n().verify_match_capitalized}">(${i18n().verify_match_capitalized}</a> ${i18n().or}
+            <a href="#" class="changeSelection" id="changeSelection">${i18n().change_selection})</a>
+        </p>
+        <input class="acUriReceiver" type="hidden" id="publicationUri" name="publicationUri" value="${publicationUriValue}"  ${flagClearLabelForExisting}="true" />
+    </div>
+
+    <p>
+        <label for="creator">${i18n().created_by}:</label>
+        <input type="text" name="createdBy" id="createdBy" label="createdBy" size="30" role="input" value="${modifiedByUriValue}">
```

Comment:
Not that it's confusing, but it looks like the code is looking for modifiedBy instead of createdBy, so can't find it and that's why that property isn't being set.